spec/factories/category.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :category do
+    title { 'Category Title' }
```

Comment:
My sugggestion is to use here sequence or `faker gem`, therefore in the tests categories will be named differently :)

Reply:
Done 👍 Now I use faker gem for creating a title :)